### ISSUE TITLE
Insufficient text contrast on multiple screens breaks WCAG accessibility compliance

### ISSUE THREAD

**Issue by niranjan-uma-shankar** [TryGhost/Ghost#24518]
### Issue Summary

Several UI elements across both light and dark modes have insufficient color contrast between text and background, causing them to fail [WCAG 2.2 Level AA contrast requirements](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html). This significantly impacts readability for users with low vision or color vision deficiencies.

## Why this matters

Accessible design ensures that the product is usable by as many people as possible, including those with visual impairments. Adhering to the WCAG contrast guidelines is not just a best practice but an essential part of building inclusive software.

## WCAG 2.2 Contrast Requirement

For normal text, the contrast ratio must be at least 4.5:1. [See official documentation](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html)

## Audit Tools Used

* Chrome DevTools > Lighthouse > Accessibility
* [WebAIM Contrast Checker](https://webaim.org/resources/contrastchecker/)

## Affected Screens

Below is a list of screens where text fails the contrast checks (level AA). Each includes visual evidence and contrast analysis results.

#### 1. `/posts`

| SCREENSHOT | CONTRAST CHECKER RESULTS
| ---- | ----
| <img width="718" height="392" alt="Image" src="https://github.com/user-attachments/assets/b8e2f94b-b483-400f-a011-6bc51bd60511" /> | <img width="738" height="649" alt="Image" src="https://github.com/user-attachments/assets/1cabb140-2527-48b0-bb92-1748b8056ee7" />

#### 2. `/posts/analytics`

| SCREENSHOT | CONTRAST CHECKER RESULTS
| ---- | ----
| <img width="689" height="622" alt="Image" src="https://github.com/user-attachments/assets/b041ba32-6bd3-4c90-90a5-33b907ca719e" /> | <img width="695" height="266" alt="Image" src="https://github.com/user-attachments/assets/34e8a8b3-8ddb-436a-9b22-408087542d91" />

#### 3. `/activitypub/explore`

| SCREENSHOT | CONTRAST CHECKER RESULTS
| ---- | ----
| <img width="327" height="912" alt="Image" src="https://github.com/user-attachments/assets/7e1f3eea-48d0-43db-9388-ce5d3ea22ce2" /> | <img width="718" height="643" alt="Image" src="https://github.com/user-attachments/assets/a7e50734-bba8-45a4-9083-721adec80d6c" />

#### 1. `/posts`

| SCREENSHOT | CONTRAST CHECKER RESULTS
| ---- | ----
| <img width="816" height="410" alt="Image" src="https://github.com/user-attachments/assets/62a150e4-fc4d-4b26-a73c-0f74d127b407" /> | <img width="716" height="666" alt="Image" src="https://github.com/user-attachments/assets/6e52088d-32f5-4c2b-8959-f2d5cd9f09cb" />

#### 2. `/posts/analytics`

| SCREENSHOT | CONTRAST CHECKER RESULTS
| ---- | ----
| <img width="1396" height="828" alt="Image" src="https://github.com/user-attachments/assets/9c5b349f-db23-45fe-8bdc-c4a91a62c251" /> | <img width="721" height="659" alt="Image" src="https://github.com/user-attachments/assets/cfae4298-fb30-410a-9075-e3377bb66e4c" />
| <img width="1364" height="733" alt="Image" src="https://github.com/user-attachments/assets/6b4644ac-638b-4ac5-9f2c-f4373c0d09a9" /> | <img width="721" height="659" alt="Image" src="https://github.com/user-attachments/assets/cfae4298-fb30-410a-9075-e3377bb66e4c" />
| <img width="1316" height="451" alt="Image" src="https://github.com/user-attachments/assets/4e8f5cea-505f-438a-b935-d4d68252e832" /> | <img width="721" height="659" alt="Image" src="https://github.com/user-attachments/assets/cfae4298-fb30-410a-9075-e3377bb66e4c" />

#### 3. `/analytics`

| SCREENSHOT | CONTRAST CHECKER RESULTS
| ---- | ----
| <img width="1395" height="939" alt="Image" src="https://github.com/user-attachments/assets/838de95e-7f64-4ce1-af26-a676c25c6bee" /> | <img width="721" height="659" alt="Image" src="https://github.com/user-attachments/assets/cfae4298-fb30-410a-9075-e3377bb66e4c" />
| <img width="1365" height="694" alt="Image" src="https://github.com/user-attachments/assets/0813fed2-3961-409e-be89-be6cfa32bfaa" /> | <img width="721" height="659" alt="Image" src="https://github.com/user-attachments/assets/cfae4298-fb30-410a-9075-e3377bb66e4c" />
| <img width="1353" height="837" alt="Image" src="https://github.com/user-attachments/assets/88fbd9c7-f84f-47b8-b1fa-11684e8e920a" /> | <img width="721" height="659" alt="Image" src="https://github.com/user-attachments/assets/cfae4298-fb30-410a-9075-e3377bb66e4c" />
| <img width="1390" height="222" alt="Image" src="https://github.com/user-attachments/assets/0dca39f7-a45e-4598-8c98-9c1d6d805640" /> | <img width="721" height="659" alt="Image" src="https://github.com/user-attachments/assets/cfae4298-fb30-410a-9075-e3377bb66e4c" />

#### 4. `/tags`

| SCREENSHOT | CONTRAST CHECKER RESULTS
| ---- | ----
| <img width="1295" height="798" alt="Image" src="https://github.com/user-attachments/assets/cd8abb1f-8687-4cad-864e-eecd0d06e123" /> | <img width="728" height="658" alt="Image" src="https://github.com/user-attachments/assets/67e0518c-708e-45fb-9a16-0b59d898f7e5" />

#### 5. `/members`

| SCREENSHOT | CONTRAST CHECKER RESULTS
| ---- | ----
| <img width="1387" height="531" alt="Image" src="https://github.com/user-attachments/assets/b7705639-83a1-4738-8ba9-1273832339e2" /> | <img width="737" height="656" alt="Image" src="https://github.com/user-attachments/assets/d85555bf-b3de-47b2-b35a-c5fd96ea3c85" />

## Recommended Next Steps

* Review all affected screens, and adjust text and background colours to meet or exceed 4.5:1 contrast ratio for normal text (3:1 for large text).
* Retest using Lighthouse and/or the WebAIM checker after fixes.

### Steps to Reproduce

1. Visit the links provided in the summary section
2. Generate Chrome devtools lighthouse accessibility report.

### Ghost Version

6.0.0-rc.0

### Node.js Version

22.13.1

### How did you install Ghost?

From source

### Database type

SQLite3

### Code of Conduct

- [x] I agree to be friendly and polite to people in this repository

**Comment by niranjan-uma-shankar** [comment]
One interesting observation is that X.com includes an option to increase contrast in their [Accessibility settings](https://x.com/settings/accessibility). This approach allows them to use standard text and background colors for most users, even if those choices don’t fully meet WCAG contrast guidelines, while still offering a higher-contrast mode for users with visual impairments.

<img width="625" height="232" alt="Image" src="https://github.com/user-attachments/assets/e8bddf05-6bb5-406e-82f1-baa804257e4c" />

**Comment by niranjan-uma-shankar** [comment]
Unless a WCAG audit has been completed, this issue is likely still valid.

[Stalebot says](https://github.com/TryGhost/Ghost/issues/24518#issuecomment-3561529848):

> that you do not post comments to keep the issue open if you are not actively working on a PR.

Just to clarify, I believe a designer's input is needed with colour and contrast values, before a PR can be opened. So, design blocks dev, IMO. I'm happy to open a PR, if inputs from the design team are available.
